src/Resources/QueueMonitorResource/Widgets/QueueStatsOverview.php

```diff
@@ -26,10 +28,14 @@ protected function getCards(): array
             ->map(fn (string $queue): int => Queue::size($queue))
             ->sum();
 
+        $totalJobs = Number::format($aggregatedInfo->count ?? 0);
+
+        $executionTime = CarbonInterval::seconds($aggregatedInfo->total_time_elapsed ?? 0)->cascade()->forHumans(short: true, parts: 3);
```

Comment:
The "parts: 3" limit the amount of units displayed, meaning that if the widget rolls over todays, the seconds will be dropped and no longer be displayed. I wanted to make sure the text wont get to long.